lib/Echidna/Transaction.hs

```diff
@@ -84,12 +87,12 @@ genTxM :: (MonadRandom m, MonadReader x m, Has TxConf x, MonadState y m, Has Gen
   => Map Addr Contract
   -> m Tx
 genTxM m = do
-  TxConf _ g maxGp t b mv <- view hasLens
+  TxConf _ g gp t b mv <- view hasLens
   genTxWith
     m
     rElem rElem                                                                -- src and dst
     (const $ genInteractionsM . snd)                                           -- call itself
-    (pure g) (inRange maxGp) mv                                                -- gas, gasprice, value
```

Comment:
is this still an issue?

Reply:
What exactly?

Reply:
gasprice, but I've read the updated description and see what's the problem